### ISSUE TITLE
[FEATURE] Limit Carrier Options for Shipping

### ISSUE THREAD

**Issue by sewcialite** [AccelerateNetworks/NumberSearch#402]
**What is the problem that this feature solves?**
Like the drop-down menu for the condition, it would be nice to have options for shipping carriers as well. Teledynamics only ships with UPS, Fedex and USPS. 

**Describe the solution you'd like**

UPS
FedEx
USPS
DHL
Other - for edge cases? 

**How will this improve revenue or reduce our costs?**
I am lazy and it makes it go faster.

**Comment by uncheckederror** [comment]
The carrier is copied from the order in TeleDynamics. We use it to generate the tracking link. Right now only UPS and FedEx will correctly generate the tracking link. Are DHL and USPS options for shipping in TeleDynamics?

**Comment by sewcialite** [comment]
I had an order I had to manually edit, and it had me type in the carrier info. For database's sake, should it still be a text box?

**Comment by uncheckederror** [comment]
> I had an order I had to manually edit, and it had me type in the carrier info. For database's sake, should it still be a text box?

Can you link me to order so I can review it?

**Comment by sewcialite** [comment]
https://ops.acceleratenetworks.com/Home/Order/4008b456-a984-4120-8f87-df128479a21f

**Comment by uncheckederror** [comment]
This order broken because of a bug in the TeleDynamics API, I have forwarded @danry25 the info he needs to send to them to fix this issue.

**Comment by uncheckederror** [comment]
We're blocked on this issue until the vendor is ready. @danry25 any update from TeleDynamics on fixing their API?

**Comment by uncheckederror** [comment]
Per my discussion with @sewcialite and @danry25 new TeleDynamics orders will be created with custom PONumber ascending from 1 which should solve this problem.

**Comment by sewcialite** [comment]
> Per my discussion with @sewcialite and @danry25 new TeleDynamics orders will be created with custom PONumber ascending from 1 which should solve this problem.

We're already to 3 or 4, so check before ordering.

**Comment by uncheckederror** [comment]
Okay, I have added a picker for the Carrier when you are manually editing a Product Item. But ideally you would just click the Link to TeleDynamics button and it will update existing Assigned Hardware with the correct tracking info if the order has a legit PONumber.

**Comment by sewcialite** [comment]
This works! Appreciated. :)
